docs/literate.jl

```diff
@@ -53,37 +62,32 @@ function preprocess(content)
     # remove VSCode `##` block delimiter lines
     content = replace(content, r"^##$."ms => "")
 
-    # adds the current version of the packages
-    append = """
-    # ### Package and system information
-    # #### Package version
-    # ```julia
-    $(chomp(replace(pkg_status, r"^"m => "# ")))
-    # ```
-    # #### Computer information
-    # ```
-    $(chomp(replace(sprint(InteractiveUtils.versioninfo), r"^"m => "# ")))
-    # ```
-    # #### Manifest
-    # To reproduce this notebook's package environment, you can [download the full Manifest.toml]($(MANIFEST_OUT)).
-    """
-    # This regex add "# " at the beginning of each line
-    # chomp removes trailing newlines
+    """ The regex adds "# " at the beginning of each line; chomp removes trailing newlines """
+    literate_format(s) = chomp(replace(s, r"^"m => "# "))
 
-    # The following gives a preview of the manifest
-    # embedded in a "spoiler", but this is unfortunately very buggy
-    # ```@raw html
+    # <details></details> seems to be buggy in the notebook, so is avoided for now
+    info_footer = """
+    # ---
+    #md # ```@raw html
     # <details>
-    # <summary> Show the full Manifest </summary>
-    # ```
-    # ```julia
-    # $(chomp(replace(manifest_status, r"^"m => "# ")))
-    # ```
-    # ```@raw html
+    # <summary><strong>Package and system information (click to expand)</strong></summary>
```

Comment:
No need to use a bold font (looks a bit strange IMO). I think it would also be nice to have separate sections for package information and system information. Ie. here just
```suggestion
    # <summary>Package versions (click to expand)</summary>
```

Reply:
I had it without the bold font before, and it looked stranger to me.

Reply:
The section I suggested multiple times would already mark the footer clearly. And additionally naturally allow to have separate summaries for packages, manifest, and system information. This seems reasonable since probably people are much more interested in the manifest or packages than the system information.

Reply:
I think most people aren't interested in any of it, and for that reason I'd rather hide it all away in a single section. We only added it at all to allow for reproducing the exact state used in docs generation without having to include the Manifest.toml in the repository (and updating it all the time).

Reply:
If one is not interested in package or system information, one can simply skip or ignore the corresponding section. I don't buy it that this is more difficult and that the output is too verbose if we include 3 additional lines (heading + 3 summaries instead of one summary). With the separate section and summaries, however, it will be much easier for anyone who is interested in any of this information to actually find it.